test/references/render/BeforeAfterFull/img1d_1_compare.txt

```diff
@@ -0,0 +1,6 @@
+- REFERENCE -------------------
+Gray{Normed{UInt8,8}}[Gray{N0f8}(0.0), Gray{N0f8}(0.102), Gray{N0f8}(0.2), Gray{N0f8}(0.298), Gray{N0f8}(0.4), Gray{N0f8}(0.502), Gray{N0f8}(0.6), Gray{N0f8}(0.698), Gray{N0f8}(0.8), Gray{N0f8}(0.902)]
+-------------------------------
+- ACTUAL ----------------------
+Gray{Normed{UInt8,8}}[Gray{N0f8}(0.902), Gray{N0f8}(0.8), Gray{N0f8}(0.698), Gray{N0f8}(0.6), Gray{N0f8}(0.502), Gray{N0f8}(0.4), Gray{N0f8}(0.298), Gray{N0f8}(0.2), Gray{N0f8}(0.102), Gray{N0f8}(0.0)]
+-------------------------------
```

Comment:
How each pixel is rendered may change from versions to versions, in that case, the test will fail, but we may just regenerate these.

Cref: https://github.com/JuliaGraphics/ColorTypes.jl/issues/202